src/seed/index.ts

```diff
@@ -0,0 +1,197 @@
+import { Payload } from 'payload'
+
+async function all_collections_empty(payload: Payload): Promise<boolean> {
+  const count_users = await payload.count({ collection: "users" });
+  const count_events = await payload.count({ collection: "events" });
+  const count_media = await payload.count({ collection: "media" });
+  return count_events.totalDocs == 0 && count_events.totalDocs == 0 && count_media.totalDocs == 0;
+}
+
+async function create_user(email: String, password: String, is_admin: boolean, payload: Payload) {
```

Comment:
Just a small note. In this case I would prefer `is_admin` to be something like `role: 'admin' | 'user'` so in function calls it is clearer what the argument does.

Read something like https://alexkondov.com/should-you-pass-boolean-to-functions/